Makefile

```diff
@@ -50,7 +50,7 @@ integrationtests: clean env preintegration
 	source .virtualenv/bin/activate
 	pip3 install -e .[TEST]
 	export IRIDA_UPLOADER_TEST='True'
-	xvfb-run --auto-servernum --server-num=1 integration-test $(branch)
+	xvfb-run --auto-servernum --server-num=1 integration-test $(branch) $(db_host) $(db_port)
```

Comment:
Is there a way to make these dash options?  So like `--db_host 128....` and `--db_port 8888`?  Without the flags these seem a bit odd here.

Reply:
I tried but it will not work as `make` will consume them thinking they are arguments for `make`. There is a work around, but when you do it then `xvfb-run` consumes them instead.